test/cppsuite/test_harness/thread_context.h

```diff
@@ -47,10 +48,12 @@ enum class thread_operation {
 /* Container class for a thread and any data types it may need to interact with the database. */
 class thread_context {
     public:
-    thread_context(std::vector<std::string> collection_names, thread_operation type, int64_t max_op,
+    thread_context(timestamp_manager *timestamp_manager, workload_tracking *tracking,
```

Comment:
Had to add the timestamp manager and tracking due to the use of static functions